scripts/run_mission1_robustness.py

```
@@ -0,0 +1,519 @@
+
+def stop_process(proc: subprocess.Popen | None) -> None:
+    if proc is None:
+        return
+    if proc.poll() is None:
+        try:
+            if os.name == "nt":
+                proc.send_signal(signal.CTRL_BREAK_EVENT)
+                time.sleep(1.0)
+            proc.terminate()
+            proc.wait(timeout=5)
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)